### PR TITLE
Dark theme

### PR DESCRIPTION
**Pull request by nomisum**

### CODE REVIEW

File: frontend/src/components/Container.vue [hunk 1]
```diff
@@ -81,6 +83,42 @@ export default class ContainerVue extends Vue {
         }
     }
 
+    #gastspieler &__header {
```
Comment (DerZade) [inline frontend/src/components/Container.vue]: This is unacceptable. We need a solution which works for all container with a custom header color.

**Review by DerZade — CHANGES_REQUESTED** [review]
~~In my opinion is [this](https://i.imgur.com/JWCnfsS.gif) is kinda ugly :D~~ Fixed in 4cbe27f

~~Also the quotation marks on retweets are barely visible ([see here](https://i.imgur.com/GEBJjgp.png))~~ Fixed in 62a9a68

File: frontend/src/components/Container.vue (hunk 1 above)
Comment (nomisum) [reply frontend/src/components/Container.vue]: something darken? idk. how to determine when to switch from dark to light?

Comment (nomisum) [reply frontend/src/components/Container.vue]: no solution possible with (s)css only imo

Comment (nomisum) [reply frontend/src/components/Container.vue]: sth like https://medium.com/better-programming/generate-contrasting-text-for-your-random-background-color-ac302dc87b4 would be needed for a generic solution supporting all colors.

Comment (nomisum) [reply frontend/src/components/Container.vue]: found no solution for quotation fill btw

Comment (DerZade) [reply frontend/src/components/Container.vue]: Should be fixed in 
b367efa

**Review by DerZade — APPROVED** [review]

**Pull request merged**